docs/installation_windows.md

```diff
@@ -1,38 +1,12 @@
-# Install LISA
-
-- [Prerequisites](#prerequisites)
-- [Notes](#notes)
-- [Install Python](#install-python)
-    - [Install Python in Linux](#install-python-in-linux)
-    - [Install Python in Windows](#install-python-in-windows)
-- [Install dependencies](#install-dependencies)
-    - [Install dependencies in Linux](#install-dependencies-in-linux)
-    - [Install dependencies in Windows](#install-dependencies-in-windows)
+# Install LISA on Windows
+
+- [Install Python on Windows](#install-python-on-windows)
+- [Install dependencies on Windows](#install-dependencies-on-windows)
 - [Clone code](#clone-code)
-- [Install Poetry](#install-poetry)
-    - [Install Poetry in Linux](#install-poetry-in-linux)
-    - [Install Poetry in Windows](#install-poetry-in-windows)
+- [Install Poetry on Windows](#install-poetry-on-windows)
+- [Verify installation](#verify-installation)
 - [FAQ and Troubleshooting](#faq-and-troubleshooting)
 
-LISA can be used to run test against the local node, or a remote node; if it is
```

Comment:
This is common part of Linux and Windows. Where is it go?
The picture is important to emphasis where the LISA is deployed. The question is asked several times by users. So it's worth to have a picture to emphasis it.